db/migrate/20231029231440_add_task_due_date_reminder_notification.rb

```diff
@@ -0,0 +1,5 @@
+class AddTaskDueDateReminderNotification < ActiveRecord::Migration[7.0]
+  def change
+    add_column :my_modules, :notification_sent, :boolean, default: false
```

Comment:
rename this to `:due_date_notification_sent`, as we might have others in the future